### PR TITLE
Update fence_aliyun.py

### PR DESCRIPTION
**Pull request by dongchen126**
1.Import region provider to enable Alibaba cloud private zone functionality which support  API endpoint access from intranet
2.Add ram role functionality to improve security which disable AccessKey ID and Access Key Secret

### CODE REVIEW

**Comment by dongchen126** [comment]
1.Import region provider to enable Alibaba cloud private zone functionality which support API endpoint access from intranet
2.Add ram role functionality to improve security which disable AccessKey ID and Access Key Secret

File: agents/aliyun/fence_aliyun.py [hunk 1]
```diff
@@ -121,12 +122,20 @@ def define_new_opts():
 		"required" : "0",
 		"order" : 4
 	}
+	all_opt["ram_role"] = {
+		"getopt": "m:",
```
Comment (oalbrigt) [inline agents/aliyun/fence_aliyun.py]: Can you update this to `":"`, and remove `-m` from the help text? I dont think it makes sense having `-m` for the `ram-role`.

**Comment by oalbrigt** [comment]
ok to test

**Comment by oalbrigt** [comment]
LGTM. Thanks.

**Pull request merged**